model/task_lifecycle.go

```diff
@@ -671,6 +673,118 @@ func doBisectStepback(ctx context.Context, t *task.Task) error {
 	return nil
 }
 
+func doBisectStepbackForGeneratedTask(ctx context.Context, generator *task.Task, generated *task.Task) error {
+	var s task.GeneratedStepbackInfo
+	if i := generator.StepbackInfo.GetGeneratedStepbackInfo(generated.DisplayName, generated.BuildVariant); i != nil {
+		// Carry over from the last task.
+		s = *i
+	} else {
+		// If this is the first iteration of stepback, we must get the initial condition (last successful passing task).
+		// If the generated task is not failed, stepback should be skipped.
+		if generated.Status != evergreen.TaskFailed {
```

Comment:
This logic is intended to stop those errors from splunk. I'm going to do some more testing to see if it does, but the errors originate because stepback reaches this block but the generated status is not failing

Reply:
I think that was happening because [this line](https://github.com/evergreen-ci/evergreen/pull/7709/files#diff-4bef865c41d6d16e1056e61806c23fb8bbc896b3739472dd60f0dd463ef5d2b9R1431) has a bug where a task can mistakenly enter stepback if it's a succeeded generated task. Once that's fixed this condition can be removed.